modules/ingress/variables.tf

```diff
@@ -19,14 +19,24 @@ variable "path" {
     {
       service_name = "nginx"
       service_port = "80"
-      path         = "/alb-terraform-created"
```

Comment:
is it ok to have path empty by default, the code seems uses this value?
will be good to have type as following:
```terraform
type = list(object({
  service_name = string
  service_port = string
  path = string
}))
```